src/context/theme/theme-context.jsx

```diff
@@ -0,0 +1,33 @@
+import { createContext, useContext, useState, useEffect } from 'react';
+
+const ThemeContext = createContext();
+
+const ThemeProvider = ({ children }) => {
+	const [theme, setTheme] = useState('light');
+	const toggleTheme = () => {
+		setTheme((currentTheme) => (currentTheme === 'light' ? 'dark' : 'light'));
+		if (theme === 'light') {
+			document.documentElement.setAttribute('data-theme', 'dark');
```

Comment:
Suggestion : Try getting rid of Vanilla JS !

Reply:
Hey, I did not get your comment "try getting rid of Vanilla JS".

Reply:
Avoid using document in your React application.

Reply:
i think he is telling not to use "document.documentElement.setAttribute"